File: testers/avro/java/generated-and-checked-in/com/example/events/consumer/CustomerOrderHandler.java
package com.example.events.consumer;

import com.example.events.CustomerOrder;
import com.example.events.header.StandardHeaders;

/** Handler interface for customer-order topic events */
public interface CustomerOrderHandler {
  /** Handle a message from the topic */
  void handle(String key, CustomerOrder value, StandardHeaders headers);
}

File: testers/avro/kotlin/generated-and-checked-in/com/example/events/consumer/CustomerOrderHandler.kt
package com.example.events.consumer

import com.example.events.CustomerOrder
import com.example.events.header.StandardHeaders

/** Handler interface for customer-order topic events */
interface CustomerOrderHandler {
  /** Handle a message from the topic */
  abstract fun handle(
    key: kotlin.String,
    value: CustomerOrder,
    headers: StandardHeaders
  )
}